docs/stable/1-client.md

```diff
@@ -94,33 +96,39 @@ To communicate between Status nodes, the [RLPx Transport
 Protocol, v5](https://github.com/ethereum/devp2p/blob/master/rlpx.md) is used, which
 allows for TCP-based communication between nodes.
 
-On top of this we run the RLPx-based subprotocol [Whisper
-v6](https://eips.ethereum.org/EIPS/eip-627) for privacy-preserving messaging.
+On top of this we run RLPx-based subprotocols, the client
+MAY use [Whisper v6](https://eips.ethereum.org/EIPS/eip-627), though the client 
+SHOULD use [Waku V1](https://github.com/vacp2p/specs/blob/master/specs/waku/waku-1.md)
+for privacy-preserving messaging.
 
-There MUST be a node that is capable of discovering peers and
-implements Whisper V6 specification.
+There MUST be a node that is capable of discovering peers and implements
+[Whisper v6](https://eips.ethereum.org/EIPS/eip-627),
```

Comment:
The way it is written now sounds like you MUST implement all 3. While it is actually only 1.

Reply:
I intentionally wrote this so that there "MUST be **a** node" that implements all 3. I presumed that if there is a network that MAY implement Whisper, Waku 0 and Waku 1, then that network must have at least 1 node that implements all messaging protocols.

The other alternative is to make the language conditional and state that "There MUST be a node that ... implements all the messaging protocols that the network supports".

This would probably require a section defining the term "messaging protocol".

Reply:
Right, I understand now, but this was not clear to me when I first read it.
Perhaps it would be better to explain that part somewhere where the bridging is also explained, or at least referenced from the vacp2p Waku spec. Perhaps this part does not belong in the client.md, not sure.

Reply:
>Perhaps this part does not even belong in the client.md, as you do not have to do this to create a Status client.

Yes, I agree with this. Although `client.md` is home to a lot of details about different types of nodes required for a Status client to operate effectively.

[I think it is necessary for a Whisper / Waku bridge spec page](https://github.com/status-im/specs/pull/114#issuecomment-625887093), although the bridge spec would be fairly small, the details of the bridge spec would not neatly belong in any of the currently existing or proposed spec pages.

It would also allow for easy deprecation when required. I don't know if that would be overkill. What do you think?

Reply:
@kdeme what is your opinion?

Reply:
Either works for me. Could also be just a subsection in the Whisper-usage.md. As it is bridging to support still Whisper. And can be removed easily then later on.